test/src/main/configurations/HTTP/ConfigurationApiListener.xml

```diff
@@ -100,7 +100,11 @@
 			<pipe name="callApiListener"
 				className="nl.nn.adapterframework.pipes.SenderPipe">
 				<sender className="nl.nn.adapterframework.http.HttpSender"
-					urlParam="url_param" methodType="post" resultStatusCodeSessionKey="statusCode">
+					urlParam="url_param" methodType="POST" resultStatusCodeSessionKey="statusCode"
+					allowSelfSignedCertificates="true"
+					verifyHostname="false"
+					authAlias="framework.api.user.alias"
```

Comment:
Please make the actual alias configurable, to support the configuration of an actual alias on WebSphere (where the alias names tend to be prefixed with the jvm name)